demo/admin/src/common/MasterMenu.tsx

```diff
@@ -28,13 +28,18 @@ const MasterMenu: React.FC = () => {
 
     return (
         <Menu variant={useTemporaryMenu ? "temporary" : "permanent"}>
-            <MenuItemGroup title={sectionScopeTitle}>
+            <MenuItemGroup title={sectionScopeTitle} drawerOpen={open}>
                 <MenuItemRouterLink
+                    showText={open}
                     primary={intl.formatMessage({ id: "menu.dashboard", defaultMessage: "Dashboard" })}
                     icon={<Dashboard />}
                     to={`${match.url}/dashboard`}
                 />
-                <MenuCollapsibleItem primary={intl.formatMessage({ id: "menu.pageTree", defaultMessage: "Page tree" })} icon={<PageTree />}>
+                <MenuCollapsibleItem
+                    showText={open}
```

Comment:
This should also work via the MenuContext

Reply:
I thought it would be more flexible like this 🤔

Reply:
You want me to change it?

Reply:
Yes, please change it (and also the drawerOpen prop). Both should work via MenuContext. We may add a prop to make the text controlled later, but I can't think of a use case now